app/assets/javascripts/modules/current-location.js

```diff
@@ -1,3 +1,4 @@
+// used by accordion component
```

Comment:
Can we rely on Static for this?

Reply:
We can in the browser but not in our spec tests 👎

Reply:
Yup. That's really sad. Hopefully will get fixed if we can move Static into a Rails engine.